baseobject/_lookup.py

```diff
@@ -584,17 +584,14 @@ def all_objects(
 
     Parameters
     ----------
-    estimator_types: class or list of classes, default=None
-
+    object_types: class or list of classes, default=None
         - If class_lookup is provided, can also be str or list of str
           which kind of objects should be returned.
         - If None, no filter is applied and all estimators are returned.
         - If class or list of class, estimators are filtered to inherit from
           one of these.
         - If str or list of str, classes ca be aliased by strings, via class_lookup.
-
```

Comment:
I think we might need the spaces added back here to render the bulleted lists correctly. But I'll iterate with that in the doc PR.